src/server.js

```diff
@@ -15,6 +15,9 @@ app.use(express.json());
 if (env.nodeEnv === 'development') {
   app.use(morgan('dev'));
   mongoose.set('debug', true);
+  app.use('/api/v1', apiRoute);
+} else if (env.nodeEnv === 'production') {
+  app.use('/v1', apiRoute);
 }
```

Comment:
hạn chế if else em, em tham khảo thử cách này

```suggestion
const isProduction = env.nodeEnv === 'production';
const isDevelopment = env.nodeEnv === 'development';

if (isDevelopment) {
  app.use(morgan('dev'));
  mongoose.set('debug', true);
}

const apiBasePath = isProduction ? '/v1' : '/api/v1';
app.use(apiBasePath, apiRoute);
```